requirements/requirements.txt

```diff
@@ -1,3 +1,3 @@
-omegaconf>=2.1.0dev13
+omegaconf==2.1.0dev13
```

Comment:
Please undo and rebase on master.
correct version is ==2.1.0dev16.

Reply:
The change to the omegaconf dependency is breaking. Pinning the version temporarily so that my code still runs. Here's the error you get with the latest version of `omegaconf`

```
[2021-01-22 17:15:15,075][HYDRA] Running command: ray exec --run-env=auto /tmp/tmpxbaa8z1i.yaml python /tmp/tmp.oza1lASz0e/_remote_invoke.py /tmp/tmp.oza1lASz0e
[2021-01-22 17:15:18,224][HYDRA] Output: Traceback (most recent call last):
  File "/opt/venv/lib/python3.8/site-packages/omegaconf/omegaconf.py", line 793, in open_dict
    OmegaConf.set_struct(config, False)
  File "/opt/venv/lib/python3.8/site-packages/omegaconf/omegaconf.py", line 483, in set_struct
    conf._set_flag("struct", value)
  File "/opt/venv/lib/python3.8/site-packages/omegaconf/base.py", line 102, in _set_flag
    self._invalidate_flags_cache()
  File "/opt/venv/lib/python3.8/site-packages/omegaconf/base.py", line 512, in _invalidate_flags_cache
    if self.__dict__["_flags_cache"] is not None:
KeyError: '_flags_cache'
```

Reply:
I wonder if this is caused by a version mismatch on the AWS instance vs your laptop; probably a good idea to double check to make sure the lib versions are consistent.

Reply:
Yes. is sounds like you have the wrong version of OmegaConf on the remote side.

Reply:
I thought of that. Made sure I was installing the same version in the docker container.

Reply:
Once I clear an impending deadline I'll have time to change this back and add the other things that have been requested. But at the moment I need things that were working to keep working.

Reply:
no worries.

Reply:
actually, let me just make this change on a private branch instead of the PR

Reply:
That would be even better :)